src/Components/Browser.JS/src/Boot.Server.ts

```diff
@@ -41,12 +51,16 @@ async function boot() {
   circuitHandlers.forEach(h => h.onConnectionUp && h.onConnectionUp());
 }
 
-async function initializeConnection(circuitHandlers: CircuitHandler[]): Promise<signalR.HubConnection> {
-  const connection = new signalR.HubConnectionBuilder()
+async function initializeConnection(configureSignalR: (builder: signalR.HubConnectionBuilder) => void, circuitHandlers: CircuitHandler[]): Promise<signalR.HubConnection> {
+  const connectionBuilder = new signalR.HubConnectionBuilder()
     .withUrl('_blazor')
-    .withHubProtocol(new MessagePackHubProtocol())
-    .configureLogging(signalR.LogLevel.Information)
-    .build();
+    .withHubProtocol(new MessagePackHubProtocol());
+
+  if (configureSignalR) {
```

Comment:
This is suspicious. It implies we're not running TypeScript in strict mode. If we were, it would not be possible for this to be null, as the type isn't nullable. Looking back, I'm surprised it allows L30 - is it implictly interpreting `configureSignalR` as type `any`?

Reply:
Ideally I'd expect there to be a type defined at the top of the file like:

```
type SignalRBuilder = (builder: signalR.HubConnectionBuilder) => void;
```

... and then this function's signature would be:

```
async function initializeConnection(configureSignalR: SignalRBuilder | null, circuitHandlers: CircuitHandler[]): Promise<signalR.HubConnection> { ... }
```

... so it's explicit about what's going on.